app/services/agent_interaction_service.py

```
@@ -227,7 +227,9 @@ async def send_broadcast_message(
                 .single() \
                 .execute()
 
-            total_agents = len(crew_response.data["agent_ids"]) if crew_response.data and crew_response.data.get("agent_ids") else 0
+            if not crew_response.data:
+                raise ValueError(f"Crew {crew_id} not found")
```

Ruff (0.14.14)

[warning] Abstract `raise` to an inner function

(TRY301)